File: Integrais - unidade 3/TrapezioSimples.java
import java.util.Scanner;

class TrapezioSimples {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double a, b;
        System.out.println("Digite o valor de a: ");
        a = sc.nextDouble();
        System.out.println("Digite o valor de b: ");
        b = sc.nextDouble();
        double h = b - a;
        double formula = (h / 2) * (f(a) + f(b));
        System.out.println("A área aproximada é: " + formula);
        sc.close();

    }

    public static double f(double x) {
        return (x * x) + (3 * x) + 2;
    }
}

File: Integrais - unidade 3/TrapezioSimples.py
def f(x):
  return 2*x

def trapezoidal_rule(a, b):
  h = b - a
  return (h / 2) * (f(a) + f(b))

# intervalo
a = 0
b = 2

area_aproximada = trapezoidal_rule(a, b)
print("A área aproximada é:", area_aproximada)
